build.gradle

```diff
@@ -91,13 +91,13 @@ repositories {
 
 dependencies {
 	compile 'com.dell.isg.smi:wsmanlib:1.0.29'
-	compile 'com.dell.isg.smi:wsmanclient:1.0.28'
+	compile 'com.dell.isg.smi:wsmanclient:1.0.17'
     compile 'com.dell.isg.smi:racadmlib:1.0.16'
-  	compile 'com.dell.isg.smi:commons-elm:1.0.76'
-	compile 'com.dell.isg.smi:commons-model:1.0.81'
+  	compile 'com.dell.isg.smi:commons-elm:1.0.33'
+	//compile 'com.dell.isg.smi:commons-model:1.0.65'
 	compile 'com.dell.isg.smi:adapter-server:1.0.34'
 	compile 'com.dell.isg.smi:adapter-chassis:1.0.12'
-	//compile project(':commons-model')
+	compile project(':smi-lib-commons-model')
```

Comment:
Do not check in with the inter-project reference, which is for development/debugging only.  Comment out and used pinned version number.